Sources/TuistCore/Graph/GraphLoader.swift

```diff
@@ -190,7 +190,7 @@ public final class GraphLoader: GraphLoading {
             }
 
         case .xctest:
-            return try platforms.map { platform in
+            return try platforms.sorted().map { platform in
                 try loadXCTestSDK(platform: platform)
             }.first
```

Comment:
Above at line 171 we have a similar approach. I think it's a very tiny bit faster to choose our platform and then load the SDK.  (suggestion created in GitHub and not verified by a compiler)

```suggestion
            return try platforms.sorted().first.map { platform in
                try loadXCTestSDK(platform: platform)
            }
```

Reply:
In practice they should be the same, and `case .sdk` also has the same code, so moving to this suggestion.
Thank you!